studio/lib/payloads/salaryAndBenefits.ts

```diff
@@ -0,0 +1,26 @@
+import { PortableTextBlock } from 'src/components/richText/RichText';
+
+interface Slug {
```

Comment:
you can use Slug from pages.ts or add global interfaces in a /payloads/gobal.ts e.g